src/harness/security_testcase.py

```diff
@@ -250,3 +250,38 @@ def _InfoCb(conn, where, ok):
       self.assertEquals(client_ssl.get_peer_finished(), None)
     finally:
       client_ssl.close()
+
+  def create_short_lived_certificate(self, client_type, cert_name, cert_expiry_time):
+    """Generates short lived certificate for SCS/SDS 17,18 & 19
+
+    The function uses the root ca, cbsd ca and proxy ca certificates generated by
+    "generate_fake_certs.sh" script and creates a short lived client certificate
+    based on the input parameters.
+
+    Args:
+      client_type: Type of the device (CBSD or DomainProxy)
+      cert_name: Certificate file name to be created. The private key shall also have
```

Comment:
Should this include the ".cert" suffix? Either way, the comment is misleading. Please clarify.

Reply:
It is   just the name for the certificate and private key without the .cert suffix.   The parameter description has been updated accordingly.